### PR TITLE
chore: update to capacitor 5

### PR DESCRIPTION
**Pull request by AlexanderMelde**
As described in issue #54.

I ran the migration script and `npm install`, however i could not fully test the plugin.

### CODE REVIEW

**Comment by lovetodream** [comment]
Thanks for the PR, I'm no longer using Capacitor so your contribution comes in very handy.

What do you mean with, "I couldn't fully test the plugin"?

File: package.json
Comment (lovetodream) [inline package.json:1]: Why did you decide to skip version 4?

**Comment by AlexanderMelde** [comment]
> Thanks for the PR, I'm no longer using Capacitor so your contribution comes in very handy.

You're welcome :) Thanks for still responding here.

> What do you mean with, "I couldn't fully test the plugin"?

I never developed a capacitor plugin before, so I simply ran the install/update commands and checked for compilation errors with the IDE, but I did not check if you have written any test cases (or wrote any myself).

> Why did you decide to skip version 4?

I saw other plugins ([those from capawesome](https://github.com/capawesome-team/capacitor-plugins/blob/main/packages/background-task/CHANGELOG.md)) using this method, so that the major version of the plugin corresponds to the major version of capacitor.

File: package.json
Comment (rengert) [reply package.json:1]: Would be great to have the same major as capacitor has

**Review by lovetodream — APPROVED** [review]

**Pull request merged**